src/test/regress/expected/gp_parallel.out

```diff
@@ -1847,6 +1847,39 @@ abort;
 --
 -- End of Test locus after eliding mtion node.
 --
+--
+-- Test outer path has Motion of parallel plan. 
+--
+begin;
+create table t1(a int, b int) with(parallel_workers=3);
+create table t2(b int, a int) with(parallel_workers=2);
+insert into t1 select i, i+1 from generate_series(1, 10) i;
+insert into t2 select i, i+1 from generate_series(1, 5) i;
+analyze t1;
```

Comment:
no need analyze ?

Reply:
Better than never, keep analyze  for stable plan.